src/Microsoft.OData.Core/DefaultServicesConfigurations.cs

```diff
@@ -0,0 +1,63 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="DefaultServicesConfigurations.cs" company="Microsoft">
+// Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.OData.Edm;
+using Microsoft.OData.Json;
+using Microsoft.OData.UriParser;
+
+namespace Microsoft.OData.Core
+{
+    /// <summary>
+    /// Extension methods for <see cref="IServiceCollection"/>.
+    /// </summary>
+    public static class DefaultServicesConfigurations
+    {
+        /// <summary>
+        /// Adds the default OData services to the <see cref="IServiceCollection"/>.
+        /// </summary>
+        /// <param name="services">The <see cref="IServiceCollection"/> to add the services to.</param>
+        /// <returns>The <see cref="IServiceCollection"/> instance itself.</returns>
+        public static IServiceCollection AddDefaultODataServices(this IServiceCollection services)
+        {

```

Comment:
I think you need to check `services` for null in this method as well, otherwise `services.AddDefaultODataServices(ODataVersion.V4)` might throw an exception?